docker/environments/dart-2.10.4/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM google/dart:2.10.4
+
+LABEL maintainer="team@appwrite.io"
+
+RUN apk add tar
+
+RUN mkdir /usr/local/src
```

Comment:
Seems like this dir already exists on Debian `/usr/local/src`, can remove this line